framework/lib/theme/components/toolbox/index.ts

```diff
@@ -31,14 +32,14 @@ export const Toolbox: ComponentMultiStyleConfig = {
       borderTopStyle: 'solid',
       borderTopColor: 'gray.100',
     },
-    body: {
+    body: merge({
       p: sizing['4'],
       flexDirection: 'column',
       w: '100%',
       h: '100%',
       maxH: `calc(100vh - ${sizing['16']} * 2)`,
       overflowY: 'scroll',
-    },
+    }, sx.body),
   }),
```

Comment:
![Screenshot 2024-07-29 at 15 29 59](https://github.com/user-attachments/assets/64c6ac96-dd64-4127-9206-b8adc84d0c74)
If sx is an empty object then this is gonna crash, I think I would do something like the pic instead
![Screenshot 2024-07-29 at 15 36 54](https://github.com/user-attachments/assets/068f5926-c814-447f-9057-a07c622a2564)